package.json

```diff
@@ -5,7 +5,8 @@
     "build": "pulp build -- --censor-lib --strict"
   },
   "devDependencies": {
-    "pulp": "^10.0.4",
+    "pulp": "^12.0.1",
+    "purescript": "^0.11.6",
```

Comment:
We intentionally don't include a purescript dependency in the package.json for core libraries since Travis is set up to always pull the latest release in instead.

Reply:
Oh I see